### DISCUSSION TITLE
Lesson 9 : Got this error message when testing on sepolia testNet

### DISCUSSION THREAD

**Discussion by AnouarBF** [Cyfrin/foundry-full-course-cu#124] [Q&A]
When I type ```forge test --fork-url $SEPOLIA_RPC_URL``` to test all the contract. It shows me this error message: 
```
[FAIL. Reason: Setup failed: Custom Error d8a3fb52:(0x8f478e8C52EccD5012a0C8B112766bAD5d67C55F)]
```
I passed the deployerKey to vm.startBrodcast() than got this error. It seems that the RaffleTest doesn't setup, before testing !

**Comment by alymurtazamemon** [comment]
@AnouarBF Please leave repository link so we can see the code.

**Reply by cromewar** [reply]
yes please!

**Comment by Aishwaria8786** [comment]
Were you able to solve the problem? 
I am facing the same thing.

**Reply by MarcosPosseB** [reply]
Hello! 

Did you solve it? I'm facing the same error. Thanks!
